core/tests/test_utils.py

```diff
@@ -2933,16 +2704,12 @@ def _send_email_to_recipients(
         Returns:
             bool. Whether the emails are sent successfully.
         """
-        bcc_emails = None
-
-        if bcc:
-            bcc_emails = bcc[0] if len(bcc) == 1 else bcc
+        bcc_emails = bcc and (bcc[0] if len(bcc) == 1 else bcc)
```

Comment:
Not sure if you're handling the "bcc=[]" case correctly? (It would have been None before, I think now it's [].)